bots/reads.ts

```diff
@@ -410,36 +412,42 @@ const handleUpdateUrl = async (url: string, ctx: ReadsContext, config: ReadsConf
   ctx.session.item.link = cleanUrl;
 
   try {
-    metadata = await fetchUrlMetadata(cleanUrl, config);
-    await ensureNonDuplicateLink(cleanUrl, config);
+    if(cleanUrl.includes('x.com')) {
+      metadata = await fetchUrlMetadata(cleanUrl, config);
+      await ensureNonDuplicateLink(cleanUrl, config);
+
+      const { description, image, title } = metadata
+      
+      ctx.session.item.title = title || '';
+      ctx.session.item.tags = defaultTagsForUrl(cleanUrl);
+
+      ctx.session.item.description
+        = (description && description.length > 500)
+          ? description.substring(0, 497) + '...'
+          : description || '';
+    
+      ctx.session.item.image_url = image || '';
+    } else {
+      const summary = await fetchUrlSummary(cleanUrl, config)
```

Comment:
Per the task instructions, If this call fails, fall back to the `metadata` description already in place on the item, send a message to the user that the AI summary call failed.

given this, you probably want to revert back to the existing `try` block logic for the metadata call, and add a second `try` block for the openai call, otherwise there will be a lot of duplicate code.